src/haz3lcore/zipper/action/Action.re

```diff
@@ -35,13 +35,19 @@ type rel =
   | Current
   | Id([@equal (_, _) => true] Id.t, Direction.t);
 
+[@deriving (show({with_path: false}), sexp, yojson, eq)]
```

Comment:
don't totally get what these actions are supposed to be. definition of what? body of what?